modules/apps/frontend-js/frontend-js-web/webpack.config.js

```diff
@@ -13,29 +13,91 @@
  */
 
 const path = require('path');
+const { container: { ModuleFederationPlugin } } = require('webpack');
 
 const PUBLIC_PATH = '/o/frontend-js-web/liferay/';
 
-module.exports = {
-	context: path.resolve(__dirname),
-	devtool: 'source-map',
-	entry: './src/main/resources/META-INF/resources/liferay/global.es.js',
-	mode: 'production',
-	module: {
-		rules: [
-			{
-				exclude: /node_modules/,
-				test: /\.js$/,
-				use: {
-					loader: 'babel-loader',
+module.exports = [
+	{
+		context: path.resolve(__dirname),
+		devtool: 'source-map',
+		entry: './src/main/resources/META-INF/resources/liferay/global.es.js',
+		mode: 'production',
+		module: {
+			rules: [
+				{
+					exclude: /node_modules/,
+					test: /\.js$/,
+					use: {
+						loader: 'babel-loader',
+					},
 				},
-			},
-		],
+			],
+		},
+		output: {
+			filename: 'global.bundle.js',
+			libraryTarget: 'window',
+			path: path.resolve('./build/node/packageRunBuild/resources/liferay/'),
+			publicPath: PUBLIC_PATH,
+		},
 	},
-	output: {
-		filename: 'global.bundle.js',
-		libraryTarget: 'window',
-		path: path.resolve('./build/node/packageRunBuild/resources/liferay/'),
-		publicPath: PUBLIC_PATH,
+	{
```

Comment:
I think mostly same comments in https://github.com/liferay-frontend/liferay-portal/pull/527/commits/3da4ee8eaf3213e3e25374b1e1f71e42a180f2c9 apply below this line